python/MooseDocs/test/gold/latex/extensions/include.tex

```diff
@@ -1,9 +1,9 @@
 
 \chapter{\label{include-extension}Include Extension}
-\par We hold these truths to be self-evident, that all men are created equal, that they are endowed by their Creator with certain unalienable Rights, that among these are Life, Liberty and the pursuit of Happiness. 
-\par We hold these truths to be self-evident, that all men are created equal, that they are endowed by 
+\par We hold these truths to be self-evident, that all men are created equal, that they are endowed by their Creator with certain unalienable Rights, that among these are Life, Liberty and the pursuit of Happiness.
+\par We hold these truths to be self-evident, that all men are created equal, that they are endowed by
 \par Header {\textasciicircum} Right of the People to alter or to abolish it, and to institute new Government\$ Footer
 \begin{quote}
-\par Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances. 
-\par A well regulated Militia, being necessary to the security of a free State, the right of the people to keep and bear Arms, shall not be infringed. 
+\par Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances.
```

Comment:
You have a bunch of TeX files that just have whitespace changes. Those shoudl be removed from this PR. I can help you with that.